fix: removing references to (previous) versions of Che in two .adoc files (#2062)
codenvy/che-docs: modules/end-user-guide/examples/che-ref_a-kubernetes-secret-as-a-file.adoc

    app.kubernetes.io/component: workspace-secret
...
----

> // The corresponding procedure is underdeveloped. Parts of this example will likely end up in the procedure. A comprehensive rewrite is needed. max-cx

Reviewer: If working on the rewrite is out of scope of this task, file an issue for the work (in GH and a corresponding one in JIRA) and link it here (the GH one), so that the comment would have some follow up.

Author: @rkratky, I already opened a JIRA issue, and that's why I've not done more rewriting in this PR - most of the example text here has to be moved to procedures. I will open a GH issue as well.

> * The following example shows you how to mount the secret as a file by including the `<url>/mount-as: file` annotation.

Reviewer: ```suggestion
* The following example shows how to mount the secret as a file by including the `<url>/mount-as: file` annotation:
```

    app.kubernetes.io/component: workspace-secret
...
----

> The following examples show more `metadata` of a secret:

Reviewer 2: As a user, I want to know why this metadata is not in the first example. When should I add or remove them?

Reviewer 2: ```suggestion
The following examples show optional `metadata` you can use with secrets for a variety of reasons:
```

> * The `annotations` must specify that the given secret is to be mounted as a file: `/mount-as: file`.

Reviewer 2: ```suggestion
* If you use `annotations`, you must always include the `<url>/mount-as: file` annotation, which mounts the secret as a file.
```

  labels:
...
----

> * The `/automount-workspace-secret` annotation with a boolean value defines the default secret mounting behavior (that can be overridden in a devfile):

Reviewer 2: ```suggestion
* To define the default behavior for mounting secrets, use the  `/automount-workspace-secret` annotation. Developers can override this default in a devfile.  This annotation takes a boolean value:
```

** `true` enables automatic mounting into all workspace containers.

> ** `false` disables the mounting process until it is explicitly requested in a devfile component using the `automountWorkspaceSecrets:true` property.

Reviewer 2: ```suggestion
** `false` disables automatic mounting into all workspace containers. To mount a secret, a user must request it in the devfile component by using the `automountWorkspaceSecrets:true` property.
```

  labels:
...
----

> * The following example shows you how to mount the `settings.xml` file at the `{prod-home}/.m2/` path of all workspace containers.

Reviewer: ```suggestion
* The following example shows how to mount the `settings.xml` file at the `{prod-home}/.m2/` path of all workspace containers:
```

  labels:
...
----

> * In the next example, the file `settings.xml` will be mounted at the `{prod-home}/.m2/` path of all workspace containers.

Reviewer 2: ```suggestion
* The following example shows you how to mount the `settings.xml` file at the `{prod-home}/.m2/` path of all workspace containers.
```

  settings.xml: __<base64 encoded data content here>__
----
+
NOTE: The `data` of the {orch-name} secret can contain several items. The names of these items must match the desired file name mounted into the container.

> * You can override the mount path of the secret for a workspace component by editing the devfile as follows: Declare a component alias for the containers in the override path. Declare an additional volume for the component. Use the name of the secret as the name of the volume. Provide the new mount path for the secret as `containerPath`. See the following example.

Reviewer: ```suggestion
* To override the mount path of the secret for a workspace component, edit the devfile as follows:
+
. Declare a component alias for the containers in the override path.
. Declare an additional volume for the component.
. Use the name of the secret as the name of the volume.
. Provide the new mount path for the secret as `containerPath`.
+
See the following example:
```

> //"volume" here means storage volume? max-cx

Reviewer: Find out before submitting the doc. Such comments in the file add confusion. (And yes, it pbbly means 'storage volume'.)

Author: @rkratky, OK. I reasoned these comments would be read and answered at the review of this merge request. And if not, I'd still be rewriting the whole file for the follow-on issue.

  settings.xml: __<base64 encoded data content here>__
----
+

> NOTE: The `data` of the {orch-name} secret may contain several items, whose names must match the desired file name mounted into the container.

Reviewer 2: ```suggestion
NOTE: The `data` of the {orch-name} secret can contain several items. The names of these items must match the desired file name mounted into the container.
```

Reviewer 2: Or maybe something like 
```
NOTE: When you create items in the of the {orch-name} secret `data` element, the names of these items must match the file name [of the secret?] you are mounting into the container.
```

> * It is possible to override the secret's mount path for specific components of the workspace by declaring an additional volume for a component with the volume's `name` matching the secret's name and with the desired mount path (`/my/new/path`) as `containerPath`, as shown next.

Reviewer 2: ```suggestion
* You can override the mount path of the secret for specific workspace components. To do this, declare an additional volume for a component. Match its `name` and `containerPath` with the mount path of the secret, as shown in the following example.
```

       containerPath: /my/new/path
   ...
----

> NOTE: With overrides, an alias declared in the components will distinguish the containers for the override, and the override path will apply only to those containers.

Reviewer 2: I'm not sure what this note is saying. I recommend addressing the user in the second person and telling them what they should expect to see and do.

IBM SG: "Tense: Write in the simple present tense as much as possible. Use past or future tense only when you cannot use present tense or it does not make sense to use present tense."

Also, "distinguish" is a cognitive process. It is typically performed by a person or animal, and is often used with "from". For example, in the active voice, "You can usually distinguish butterflies from moths by their greater size and variety of color."   Or, in the passive voice, "Butterflies and moths can typically be distinguished from each other by their size and variety off color." To make this previous example sound more antique and literary, I can rewrite it as,  "Their size and variety of color will distinguish butterflies from moths." However, the structure of this last example is not effective in technical writing.

Author: This is work in progress. I committed these changes, but I have a different draft open.
By the way, I'm only planning fixing this sentence, because I opened a new issue for this:
https://issues.redhat.com/browse/RHDEVDOCS-3166

Author: Would you mind having a second look? I double-checked the relevant statements within the original file, so it should be factually correct now.

       containerPath: /my/new/path
   ...
----

> NOTE: For such overrides, you must declare an alias for the components to distinguish the containers which belong to them and to apply the override path exclusively to those containers.

Reviewer 2: Use `With overrides,` instead. This run-on sentence is hard to decipher. Here, the meaning of "them" is ambiguous and the `to...to...to...to` construction is hard to interpret. Please break this into multiple simple statements.

Author: I get it. It's only two infinitives in parallel, but those are difficult to follow, so I need to improve readability here. I'll see how I can split this into two sentences and possibly rearrange them as well.

Reviewer 2: There are other issues here, which we'll discuss after you make the updates.
